gilbreth_gazebo/include/gilbreth_gazebo/plugins/ConveyorBeltPlugin.hh

```diff
@@ -160,7 +160,7 @@ namespace gazebo
     private: math::Angle limit;
 
     /// \brief Maximum linear velocity of the belt.
-    private: const double kMaxBeltLinVel = 0.2;
+    private: double kMaxBeltLinVel = 0.4;
```

Comment:
Does this means that the maximum velocity of the conveyor belt is 0.4 m/sec? I tried this value with 100% power. The length between the laser beam and the center line is 2.8 meter. Total time for a part to move from the laser to the middle is approximately 3 second, which shows that the velocity of the conveyor is around 1m/sec. 
![screenshot from 2018-02-27 14-07-30](https://user-images.githubusercontent.com/16554844/36750675-255583ee-1bcc-11e8-9f26-70f73565c4e1.png)
![screenshot from 2018-02-27 14-07-13](https://user-images.githubusercontent.com/16554844/36750676-255f2778-1bcc-11e8-877a-45518b7869c2.png)

Reply:
I believe so
However, I did lower the power so that the overall conveyor speed remained the same.  Let me know if this is not the case.

Reply:
Thank you. I didn't notice that it can be configured in the .sdf file.

Reply:
Yes, sorry It was not something that I did a very good job at advertising.